README.rst

```diff
@@ -128,6 +118,16 @@ If you want to work interactively *inside* the container, replace the name of th
 You are then the root user in the container. Octopus was compiled in ``/opt/octopus*``. There are also some trivial examples in ``/opt/octopus-examples``.
 
 
+Information for developers: available architectures
+---------------------------------------------------
+
+The DockerHub images are available for x86 (AMD64) and M1/M2 (ARM64)
+architectures. Docker will download the correct one automatically. (You can use
+``docker inspect octimate | grep Arch`` to check the architecture
+for which you have the image ``octimage`` available on your machine,
```

Comment:
You could explicitly mention the publically available docker image name to avoid confusion,

```suggestion
``docker inspect fangohr/octopus | grep Arch`` to check the architecture
for which you have the image available on your machine,
```

Reply:
optional. I would merge the MR even without this change.